docs/source/algorithms/D7YIGPositionCalibration-v1.rst

```diff
@@ -60,10 +62,11 @@ Usage
    Load('ILL/D7/396442_396831.nxs', OutputWorkspace='intermediateWavelengthScan')
    approximate_wavelength = '4.8' # Angstrom
    D7YIGPositionCalibration(InputWorkspace='intermediateWavelengthScan', ApproximateWavelength=approximate_wavelength,
-                               YIGPeaksFile='D7_YIG_peaks.xml', CalibrationOutputFile='test_intermediateWavelength.xml',
-                               MinimalDistanceBetweenPeaks=1.5, BankOffsets="-4,-4,0",
-                               FittingMethod='Individual', FitOutputWorkspace='intermediateWavelength')
-		       
+                            YIGPeaksFile='D7_YIG_peaks.xml', CalibrationOutputFile='test_intermediateWavelength.xml',
+                            MinimalDistanceBetweenPeaks=1.5, BankOffsets=[-4,-4,0],
+			    MaskedBinsRange=[-50, -25, 15],
```

Comment:
Make sure these ahve the right indentation.

Reply:
Well spotted. This is a little bit strange, as they are all aligned on my local machine and were edited with emacs. When I open this through gedit, they also look properly aligned. I am going to correct it from github for now but need to find a source of this issue.

Reply:
i agree it looks like there's a wrong indentation, however when I open this file in emacs, or view or edit it on github, these lines seem properly aligned and indented. That's a bit strange

Reply:
You were using tabs for some lines and spaces for others.

Reply:
Oh, right. The most recent commit should've fixed the issue.